### PR TITLE
Mobile-Menu-With JS

### PR DESCRIPTION
**Pull request by Kumilachew-g**
I have done the following tasks.
 1. Added hamburger in the HTML navbar part
 2. Added styling for mobile menu
 3. Added JS function in the mobile menu
 4. Added HTML, CSS, and JS linter dependencies

Tasks look like

image

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -25,29 +26,121 @@ ul {
...
 }
 
-nav {
+a {
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: var(--sub2k);
+}
```
Comment (Ugboaja-Uchechi) [inline css/style.css:46]: - [ ] Kindly make sure your mobile menu design is spaced as close as possible to the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: script.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  hamburger.classList.toggle("active");
+  navMenu.classList.toggle("active");
+});
+
+document.querySelectorAll(".nav-link").forEach((n) =>
+  n.addEventListener("click", () => {
+    hamburger.classList.remove("active");
+    navMenu.classList.remove("active");
+  })
+);
```
Comment (Ugboaja-Uchechi) [inline script.js:14]: - [ ] Please make sure when the user clicks on any of the options in the mobile menu it takes them to the appropriate part of the webpage 🙏 as stated in the project requirement.

20220719-163158_capture

File: script.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+const hamburger = document.querySelector(".hamburger");
+const navMenu = document.querySelector(".nav-menu");
```
Comment (Ugboaja-Uchechi) [inline script.js:2]: - [ ] Please make sure you fix all linter errors 🙏

**Review by mwanawabangona — APPROVED** [review]
Hi @Kumilachew-g ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implimenting the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**